### ISSUE TITLE
missing whitespace in ncml-output

### ISSUE THREAD

**Issue by heikoklein** [metno/fimex#25]
fimex-1.5.0 is omitting all whitespace in the ncml-output for the shape and is then unable to read the ncml-files back.

From ```fimex --input.file=snap.nc --input.printNcML```

```
<variable name="time_of_arrival" type="float" shape="xytime" >
```
should be
```
<variable name="time_of_arrival" type="float" shape="x y time" >
```

**Comment by heikoklein** [comment]
Just testet 1.5.1 on ubuntu 18.04 and the bug still exists.

**Comment by alexander-buerger-met-no** [comment]
Yes, your report was about an hour too late to include a fix in 1.5.1

Obviously there is a regression, and an opportunity to add a a test.

**Comment by alexander-buerger-met-no** [comment]
Closed with commit ef6599c5aa.
